src/modules/users/components/GasStation/GasStationContent/GasStationContent.css

```diff
@@ -16,3 +18,16 @@
   display: flex;
   align-items: center;
 }
+
+@media screen and query700 {
+  .main {
+    width: calc(100vw - 28px);
+    border-radius: 5px;
+    background-color: var(--grey-blue-2);
+    box-shadow: 0px 2px 11px var(--drop-shadow);
+  }
+
+  .transactionsContainer div[class*="GroupedTransaction_description"] {
```

Comment:
I'm really not comfortable using this class name as the selector, since this is name is from post-processing so either a webpack update or postcss loader might change the format, at which point this will break.

We need to find an alternative for this.

How about just selecting the `nth` class this `div` renders as and just go from there ?